py_mmd_tools/check_mmd.py

```diff
@@ -122,7 +122,10 @@ def check_vocabulary(xmldoc):
                     }
     ok = True
     for element_name, f_name in vocabularies.items():
-        elems_found = xmldoc.findall('./{*}' + element_name)
+        if f_name == 'use_constraint_type':
+            elems_found = xmldoc.findall('./{*}' + element_name + '/{*}identifier')
```

Comment:
OK, this is only applicable for `use_constraint_type`?

Reply:
Yes, it's the only one that is actually a sub element. the others are directly the element. For now, only 4 attributes have controlled vocabularies implemented in pythesint, so I don't think it's worth it spending time to make it more generic.